sechub-webui/src/test/java/com/mercedesbenz/sechub/webui/SecHub/WebUI/system/SecHubWebUiApplicationTests.java

```diff
@@ -10,13 +10,26 @@
 import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.web.reactive.server.WebTestClient;
 
+import com.mercedesbenz.sechub.webui.CredentialService;
 import com.mercedesbenz.sechub.webui.SecHubServerAccessService;
 
-@SpringBootTest(webEnvironment = WebEnvironment.DEFINED_PORT, properties = { "server.port=8042", "management.server.port=9042", "sechub.userid=user",
-        "sechub.apitoken=example" })
+/*
+ * This test launches a real HTTP Server (Netty) during the test.
+ *
+ * The RANDOM_PORT setting will the start the test application on an available port on the system.
+ *
+ * The operating system is responsible for allocating the port and it is guaranteed to be available (source: https://stackoverflow.com/a/48923117).
+ * As a result, there should never be any conflict with the ports, even if tests are running in parallel.
+ *
+ * The random port is injected into the WebTestClient by default.
+ * In addition, one can get the port using the <code>@LocalServerPort<code> annotation.
+ */
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
 @AutoConfigureWebTestClient
+@ActiveProfiles("test")
 class SecHubWebUiApplicationTests {
```

Comment:
I like the term SystemTest.

Reply:
I will rename it to SpringBootTest, because this is the definition in the tec doc: https://mercedes-benz.github.io/sechub/latest/sechub-techdoc.html#unit-tests-with-spring-context